applications/GeoMechanicsApplication/custom_elements/steady_state_Pw_piping_element.cpp

```diff
@@ -0,0 +1,358 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+
+// Application includes
+#define _USE_MATH_DEFINES
+#include "custom_elements/steady_state_Pw_piping_element.hpp"
+#include <math.h>
```

Comment:
It would be better if you could use `#include <cmath>` here. <math.h> is a deprecated C header.